home/_templates/index.html

```diff
@@ -191,6 +191,17 @@
             </div>
             <div class="big_title_txt_separ background_color"></div>
          </div>
+         <div class="big_title">
+            <div class="big_title_txt montserrat_font">Sponsors</div>
+            <div class="text_und_big_title">
+              <div class="text_und_big_title">
+                 Stickermule kindly provides PROS Stickers please check them out for any sticker needs!

```

Comment:
```suggestion
                 Sticker Mule kindly provides PROS high-quality stickers&mdash;please check them out for any sticker needs!
```

from the press page (plus some other language changes:
![image](https://user-images.githubusercontent.com/6646541/75717899-06fd2e80-5ca0-11ea-8ec3-cbcb89b438e8.png)